app/src/main/java/org/p2p/wallet/updates/handler/SolanaAccountUpdateHandler.kt

```diff
@@ -30,12 +26,9 @@ class SolanaAccountUpdateHandler(
             .result
             .value
 
-        tokensRepository.updateUserToken(
-            newBalanceLamports = response.lamports,
-            mintAddress = Constants.WRAPPED_SOL_MINT.toBase58Instance(),
-            publicKey = tokenKeyProvider.publicKey.toBase58Instance()
-        )
-        Timber.tag(TAG).d("New SOL balance received, amountInLamports=${response.lamports}, data = $data")
+        userTokensInteractor.updateSolanaBalance(newBalanceInLamports = response.lamports)
```

Comment:
updateSolBalance / updateSolTokenBalance

Reply:
Why you don't like Solana but Sol?

Reply:
we have findSolOrThrow and some other places i can't remember. and also seems to me Solana is more like a network, and Sol is more like a token. 
and Sol also is shorter, and i know you love shorter names :)